circle.yml

```diff
@@ -1,13 +1,24 @@
 machine:
   xcode:
-    version: "6.3.1"
+    version: "8.1.0"
+  environment:
+    FASTLANE_SKIP_UPDATE_CHECK: true
+    BUNDLE_PATH: vendor/bundle
+    UPLOAD_IOS_SNAPSHOT_BUCKET_NAME: eidolon-ci
 dependencies:
+  cache_directories:
+    - vendor/bundle
+    - ~/.cocoapods/repos
   override:
```

Comment:
Thanks for the tips, going to be tricky not to override dependencies since we rely on `fastlane` to set up CocoaPods-Keys keys. I'll read up on Circle's docs 👍